Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +13,324 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (IsStringArrayNullOrEmpty(peopleAndBalances))
+                return "N/A.";
+
+            //Convert to Person-Object
+            Person[] people = DataToObject(peopleAndBalances);
+
+            //Find highest value and number of people with same balance
```

Comment:
Without understanding OOP, it might be really hard to wrap your head around this, but in near future, starting from chapter 2, blocks like these should really be extracted to functions and multiple values should be returned in objects.